purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1237,34 +1237,35 @@ class Purchases internal constructor(
     }
 
     private fun getSkuDetails(
-        skus: Set<String>,
+        productIds: Set<String>,
```

Comment:
this rename is kinda the only reason i wouldn't merge this right into main...changing from skus makes this more generic for both BC4 and BC5